examples/ui-demo/README.md

```diff
@@ -1,8 +1,40 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
-## Getting Started
+# Getting Started
 
-First, run the development server:
+## Setup your Enviornment
```

Comment:
Nit: typo "enviornment" -> "environment", Also "your" should be capitalized in titles, it's only articles and prepositions that aren't.